CommandsManual.md

```diff
@@ -447,6 +572,22 @@ Changes default auth account.
 | --output | -z | No | Yes | Adjusts the [command's output](#command-output) |
 | --help | -h | No | No | Displays a description of the command. Ignores any other options |
 
+#### Auth Github ####
```

Comment:
Auth Github --> Github Auth

Reply:
This one is correct (the command is `impt auth github`)